samples/nrf9160/modem_shell/README.rst

```diff
@@ -1135,6 +1135,37 @@ For example:
 
    west build -p -b nrf9160dk_nrf9160_ns -- -DOVERLAY_CONFIG=overlay-rtt.conf
 
+LwM2M support
```

Comment:
Question, so this can be used in generic ways? What about the commands? are there certain limitations and tight couplings with custom implementation?

Secondly, maybe worth to add link for the instructions, i.e. how to  setup the server side. It seems that lwm2m client sample is having some instructions.

Reply:
The LwM2M support is generic as such, however the location assistance uses proprietary objects and can be currently used only with AVSystem's Coiote server. mosh should be able to connect to other servers as well, but the functionality will be limited.

I'm adding a note about this in the documentation and a link to instructions how the server can be configured.